### PR TITLE
add v_bank case

### PR DESCRIPTION
**Pull request by coocood**
### What problem does this PR solve? 

Add an extensible v_bank case

### What is changed and how does it work?

Add an extensible v_bank case

### Check List 

Tests 

 - E2E test

Code changes

 - Has Go code change

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
add v_bank case
```

### CODE REVIEW

File: cmd/util/suit.go [hunk 1]
```diff
@@ -301,3 +301,27 @@ func nemesisGeneratorNames(gens []core.NemesisGenerator) (names []string) {
...
+	IP   string
+	Port int32
+}
+
+// NewLocalProvisioner creates a LocalProvisioner
+func NewLocalProvisioner(ip string, port int32) *LocalProvisioner {
+	return &LocalProvisioner{IP: ip, Port: port}
+}
+
+// TearDown implements the Provisioner interface.
```
[the remark this answers is not in the document]
Comment (mahjonp) [reply cmd/util/suit.go]: @coocood I added a `LocalClusterProvisioner` before, you could use it https://github.com/pingcap/tipocket/blob/46f18d3af7dd46884be45e87fc0cff1f1c099090/pkg/cluster/local_cluster.go#L27

**Comment by mahjonp** [comment]
I added a `LocalClusterProvisioner` on https://github.com/pingcap/tipocket/pull/151 before, you can use it on debug.

File: tests/vbank/v_bank.go [hunk 2]
```diff
@@ -0,0 +1,839 @@
...
+	CreateResult   *CreateResult
+	DeleteResult   *DeleteResult
+	TransferResult *TransferResult
+	OK             bool
+	Err            string
+}
+
+// IsUnknown implements UnknownResponse interface
+func (r *Response) IsUnknown() bool {
+	return false
```
Comment (mahjonp) [inline tests/vbank/v_bank.go]: If always return false, how to resolve the situation when client doesn't known whether the request is successful?

Comment (coocood) [reply tests/vbank/v_bank.go]: By checking the transaction status in `v_bank_txn_status` table.

**Pull request merged**